config/locales/en.yml

```diff
@@ -655,6 +655,9 @@ en:
     python: Python
     rest: REST
 
+  oidc:
+    authorization_flow_warning: "Any changes to the authorization flow will only be applied to new applications; existing applications won't be updated."
```

Comment:
I'd rephrase adding `default values` => that would work when we will do auth flow per application

```suggestion
    authorization_flow_warning: "New application will take default values from this authorization flow ; changing it will not update existing applications."
```

Reply:
I don't understand the word 'default' in this context.

Reply:
ping @hallelujah

Reply:
we can change the phrase for CR1…lets just merge

Reply:
```suggestion
    authorization_flow_warning: "Any changes to the authorization flow will only be applied to new applications; existing applications won't be affected."
```

Reply:
Thanks. Based on Hery's feedback I'm now considering:

> Any changes to the authorization flow of this service will only be applied to new applications; existing applications will continue to use the original flow.